README.md

```diff
@@ -1,2 +1,26 @@
 # wg-ui
-Responsible for AMP's visual components &amp; interactions and AMP's overall user experience (including published guidelines for AMP UX). The UI WG also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility. Facilitator: @aghassemi
+**wg-ui** [Working Group](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#working-groups) is responsible for AMP's visual components &amp; interactions and AMP's overall user experience (including published guidelines for AMP UX). The UI Working Group also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility. Facilitator: @aghassemi
+
+# Members
+- @aghassemi - Ali Ghassemi (Facilitator)
+- @alanorozco - Alan Orozco
+- @andrewwatterson - Andrew Watterson
+- @cathyxz - Cathy Zhu
+- @cvializ - Carlos Vializ
+- @nainar - Naina Raisinghani
+- @sparhami - Sepand Parhami
```

Comment:
Should include Jonathan?

Reply:
good point. I am following up with him. will add later